### PR TITLE
Add electric v9.07

### PR DESCRIPTION
**Pull request by KeenWill**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/electric.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  sha256 "3e383f673183265700e0c13a5f7b1d3348247465c3d8817c845d62e4c92d86ce"
+
+  url "https://ftp.gnu.org/pub/gnu/electric/electricBinary-9.07.jar"
+  name "Electric VLSI Design System"
+  desc "Sophisticated electrical CAD system for the design of integrated circuits"
+  homepage "https://www.gnu.org/software/electric/electric.html"
+
+  container type: :naked
+
+  app "electricBinary-9.07.jar"
```
Comment (vitorgalvao) [inline Casks/electric.rb:12]: Please add a [`zap`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), if relevant to this software. See [`homebrew-createzap`](https://github.com/nrlquaker/homebrew-createzap) for an external project which can help.

Comment (KeenWill) [reply Casks/electric.rb:12]: I don't think there is anything to zap here because this just downloads a jar and puts it into the applications folder

Comment (vitorgalvao) [reply Casks/electric.rb:12]: The `zap` is to get rid of files that are created in normal usage of the app, such as preferences and configuration files.

Comment (KeenWill) [reply Casks/electric.rb:12]: No extra files are created in the process that would have to be deleted

**Review by vitorgalvao — DISMISSED** [review]

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**